chapters/inheritance.tex

```diff
@@ -1150,6 +1151,7 @@ \subsection{Annotations for Redeclaration and Modification}\label{annotation-cho
 \end{lstlisting}
 \end{example}
 \section{Selective Model Extension}\label{selective-model-extension}
+\index{break@\robustinline{break}}\index{deselection}
```

Comment:
Qualify index entry, and customary empty line for source code readability:
```suggestion

\section{Selective Model Extension}\label{selective-model-extension}\index{break@\robustinline{break}!deselection}\index{deselection}
```

Reply:
I would also have preferred to have the index entry for `break` located closer to the first mention of `break` in the section.  It makes it easier to quickly find out the role of `break` in this context, which helps determine if this is the entry one is really interested in.